man1/aur-fetch.1

```diff
@@ -0,0 +1,38 @@
+.TH AUR-FETCH 1 2018-02-14 AURUTILS
+.SH NAME
+aur-fetch \- fetches packages from a location
+
+.SH SYNOPSIS
+.SY aur
+.B fetch
+.OP -L log_dir
+.OP -t
+.YS
+
+.SH DESCRIPTION
+\fBaur-fetch\fR accepts package names from stdin, and attempts to
```

Comment:
The new iteration of `aur-fetch` always takes full URIs, both for `git` repos and `tar` archives.